internal/service/vmservice/utils.go

```diff
@@ -55,14 +57,51 @@ func IPAddressWithPrefix(ip string, prefix int) string {
 	return fmt.Sprintf("%s/%d", ip, prefix)
 }
 
-// extractNetworkModelAndBridge returns the model & bridge out of net device input e.g. virtio=A6:23:64:4D:84:CB,bridge=vmbr1.
-func extractNetworkModelAndBridge(input string) (string, string) {
-	re := regexp.MustCompile(`([^=,]+)=([^,]+),bridge=([^,]+)`)
+// extractNetworkModel returns the model out of net device input e.g. virtio=A6:23:64:4D:84:CB,bridge=vmbr1,mtu=1500.
+func extractNetworkModel(input string) string {
```

Comment:
Suggest

```go
func extractNetworkModel(input string) string {
	re := regexp.MustCompile(`([^=,]+)=([^,]+),bridge=([^,]+)`)
	matches := re.FindStringSubmatch(input)
        if len(matches) > 1 {
		return matches[1]
	}
	return "virtio"
}
```

Reply:
Update:

```
func extractNetworkModel(input string) string {
	re := regexp.MustCompile(`([^,=]+)(?:=[^,]*)?,bridge=([^,]+)`)
	matches := re.FindStringSubmatch(input)
	if len(matches) >= 2 {
		return matches[1]
	}
	return ""
}
```

Reply:
why have the bridge part of this regex? the bridge is not part of the network model

Reply:
Yeah, but since the wording bridge is always in the net-config.
So, using it in the regex.
it will return the first match. so the model.